tests/cucumber/features/step_definitions/accounts_steps.coffee

```diff
@@ -0,0 +1,66 @@
+do ->
+  'use strict'
+
+  _ = require('underscore')
+
+  module.exports = ->
+
+    url = require('url')
+
+    @Given /^I am a new user$/, ->
+      @server.call('reset')
+
+    @When /^I navigate to "([^"]*)"$/, (relativePath, callback) ->
+      @client
+        .url(url.resolve(process.env.ROOT_URL, relativePath))
+        .call(callback)
+
+    @Then /^I should see content "([^"]*)"$/, (text, callback) ->
+      @client
+        .waitForVisible('body *')
+        .getHTML 'header', (error, response) ->
+          assert response.toString().match(text)
+        .call callback
+
+    @Then /I am( not)? logged in/, (amNot, callback) ->
+      @browser
+        .execute((->
+          Meteor.userId()
+        ), (err, ret) ->
+          assert.ifError(err)
+          if amNot
+            assert.equal(ret.value, null, 'Authenticated')
+          else
+            assert(ret.value, 'Not authenticated')
+        ).call(callback)
+
+    @When 'I open the account modal', (callback) ->
+      @browser
+        .click('.sign-in', assert.ifError)
+        .waitForExist('.accounts-modal.modal.in')
+        .call(callback)
+
+    @When 'I fill out the new account form', (callback) ->
+      @browser
+        .click('#at-signUp')
+        .waitForExist('#at-field-password_again')
+        .setValue('#at-field-email', 'test' + Math.floor(Math.random() * 100) + '@user.com')
```

Comment:
Do we need to use a random email? If we're resetting the users?

If we're relying on it to be unique, it won't always work.